src/main/java/edu/mit/dos/object/ObjectService.java

```diff
@@ -57,19 +57,28 @@ public String create(@RequestParam("handle") String handle,
         digitalObject.setUpdateDate(new Date());
         digitalObject.setMetadataSource(metadataSystem);
         digitalObject.setMetadataSourceUrl(sourceSystem);
+        digitalObject.setTitle(title);
 
         final DigitalObject persistedObjected = objectJpaRepository.save(digitalObject);
 
         logger.debug("Persisted:{}", persistedObjected.getOid());
 
-        return "ok";
+        return String.valueOf(persistedObjected.getOid());
     }
 
 
     @RequestMapping(value = "/object", method = RequestMethod.GET)
     public @ResponseBody
-    String getObject(@RequestParam("oid") String objectId) {
-        return "ok";
+    DigitalObject getObject(@RequestParam("oid") String oid) {
+        if (objectJpaRepository == null){
+            logger.debug("Error - object repository not found:{}", oid);
+        }
+        final DigitalObject retrievedDigitalObject = objectJpaRepository.findByOid(Long.valueOf(oid));
+        if (retrievedDigitalObject == null){
+            logger.debug("Error - digital object not found:{}", oid);
+        }
+        logger.debug(retrievedDigitalObject.toString());
```

Comment:
This line should have the oid information, as before.

Reply:
Which line? 80 should have the oid because it is exported as a part of toString()

Reply:
You're right about that.